mlflow/transformers/signature.py

```diff
@@ -90,7 +90,7 @@ def infer_or_get_default_signature(
     For signature inference in some Pipelines that support complex input types, an input example
     is needed.
     """
-    if example:
+    if example is not None:
```

Comment:
note: This is required to allow signature inference with pandas array (the original line raises error of ambiguous boolean casting)